crates/binjs_generate_library/src/lib.rs

```diff
@@ -1,3 +1,6 @@
+//#[macro_use]
+extern crate binjs_io;
```

Comment:
Do you need the dependency outside of generated code?

Reply:
Looks like it was unnecessary.